go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/cloudflare/circl
 
 go 1.12
 
-require golang.org/x/sys v0.0.0-20190602015325-4c4f7f33c9ed
+require (
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	golang.org/x/sys v0.0.0-20190602015325-4c4f7f33c9ed
```

Comment:
@armfazh You're ware your change pulled in `/x/crypto`?  I don't think it's a big issue.